frontend/src/components/menu/MessageIndicator.test.ts

```diff
@@ -3,9 +3,9 @@ import { describe, it, expect, beforeEach, afterEach } from 'vitest'
 
 import MessageIndicator from './MessageIndicator.vue'
 
-describe('LogoImage', () => {
+describe('MessageIndicator', () => {
   const Wrapper = () => {
-    return mount(MessageIndicator)
+    return mount(MessageIndicator, { props: { numberOfMessages: 1 } })
```

Comment:
How about:
```suggestion
  const Wrapper = (opts={ props: { numberOfMessages: 1 } }) => {
    return mount(MessageIndicator, opts)
  }
```

And then calling it from the test instead of the `before` hook.